run/archiveloop

```diff
@@ -480,7 +480,8 @@ log "Starting archiveloop..."
 
 mount_all_snapshots
 
-if [[ -z "${SNAPSHOTS_DISABLED}" ]]; then
+if [ "$SNAPSHOTS_ENABLED" = "true" ]
```

Comment:
this should be either be
```
if [ "${SNAPSHOTS_ENABLED:-true}" = "true" ]
```
(and below as well)  or there should be a line like:
```
SNAPSHOTS_ENABLED=${SNAPSHOTS_ENABLED:-true}
```
after line 45 above, to handle the case where SNAPSHOTS_ENABLED isn't set in the config file for some reason.

Reply:
good call i assumed since i defaulted it to true int he configFile it wouldnt be needed. but can never be too defensive.